readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -478,29 +478,6 @@ def test_list_user_created_latest_and_stable_versions_in_default_branch_choices(
             ],
         )
 
-    def test_commit_name_not_in_default_branch_choices(self):
```

Comment:
this test wasn't needed, we already test that only an exact list of the expected branches is included above.